### PR TITLE
Add models and rails admin config for new exercise.

### PR DESCRIPTION
**Pull request by eschlange**
* Added three new models to be used in a new exercise.
* Updated existing models by adding associations.
* Added migrations for the new models.
* Added validation specs for the new models.
* Added rails admin configuration for creating exercises.

[#105691206] [#105691738]

### CODE REVIEW

File: spec/models/multisyllabic_card_spec.rb [hunk 1]
```diff
@@ -0,0 +1,15 @@
+require "rails_helper"
+
+RSpec.describe MultisyllabicCard, type: :model do
+  describe "MultisyllabicCard creation" do
```
Comment (michaelwehrley) [inline spec/models/multisyllabic_card_spec.rb]: probably don't need the word "MultisyllabicCard" here or "MultisyllabicDeck" in the other spec...

File: db/migrate/20151015135853_create_multisyllabic_card.rb [hunk 2]
```diff
@@ -0,0 +1,18 @@
+class CreateMultisyllabicCard < ActiveRecord::Migration
+  def change
+    create_table :multisyllabic_cards do |t|
+      t.string :word, null: false
+      t.string :phonetic_transcription, null: false
+      t.integer :multisyllabic_deck_id
```
Comment (michaelwehrley) [inline db/migrate/20151015135853_create_multisyllabic_card.rb]: do you want `null: false`?

File: db/migrate/20151016140313_create_multisyllabic_answer.rb [hunk 3]
```diff
@@ -0,0 +1,14 @@
+class CreateMultisyllabicAnswer < ActiveRecord::Migration
+  def change
+    create_table :multisyllabic_answers do |t|
+        t.integer :multisyllabic_card_id
```
Comment (michaelwehrley) [inline db/migrate/20151016140313_create_multisyllabic_answer.rb]: `null: false`?

File: db/migrate/20151015135853_create_multisyllabic_card.rb (hunk 2 above)
Comment (eschlange) [inline db/migrate/20151015135853_create_multisyllabic_card.rb]: Cards can be created without a deck, then later be added to a deck.

File: spec/models/multisyllabic_card_spec.rb (hunk 1 above)
Comment (eschlange) [inline spec/models/multisyllabic_card_spec.rb]: :knit_pick

**Comment by michaelwehrley** [comment]
Do you want any `dependent: :destroy` calls for the `has_many` relationships?  In addition, are there any ids that shouldn't be null.  If so, I would add similar migrations to below:
```
def change
  add_foreign_key :multisyllabic_decks, : multisyllabic_cards
end
```

File: db/migrate/20151016140313_create_multisyllabic_answer.rb (hunk 3 above)
Comment (eschlange) [inline db/migrate/20151016140313_create_multisyllabic_answer.rb]: Good point. Adding it now.

**Comment by eschlange** [comment]
@michaelwehrley No, we ran into this issue before, when we were developing the audio exercises. There was a functional requirement to not destroy associated cards for the decks.

File: app/models/multisyllabic_card.rb [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+  has_many :responses
+  has_many :multisyllabic_answers_answer
+
+  validates :word,
+            :phonetic_transcription,
+            presence: true
+
+  has_attached_file :audio_sample
+  validates_attachment_size :audio_sample, :less_than => 30.megabytes,
+                            :message => 'filesize must be less than 10 MegaBytes'
```
Comment (michaelwehrley) [inline app/models/multisyllabic_card.rb]: are we using single quotes for this project?

File: app/models/response.rb [hunk 5]
```diff
@@ -3,5 +3,6 @@
 class Response < ActiveRecord::Base
   belongs_to :flash_card
   belongs_to :audio_card
+  belongs_to :multisyllabic_card
 	belongs_to :user
```
Comment (michaelwehrley) [inline app/models/response.rb]: I know you didn't do this, but could you un-indent?  Thanks!

File: app/models/multisyllabic_card.rb (hunk 4 above)
Comment (eschlange) [inline app/models/multisyllabic_card.rb]: We are using both. Rubocop is waiting in the shadows.

File: config/initializers/rails_admin.rb [hunk 6]
```diff
@@ -205,6 +205,65 @@ def email
     end
   end
 
+  config.model 'MultisyllabicCard' do
```
Comment (michaelwehrley) [inline config/initializers/rails_admin.rb]: looks good :+1:

File: app/models/multisyllabic_card.rb (hunk 4 above)
Comment (michaelwehrley) [inline app/models/multisyllabic_card.rb]: :shadows

**Comment by michaelwehrley** [comment]
After updates, please go ahead and merge.  Thanks! :smiling_imp:

**Pull request merged**